data/tokenizer/rwkv_tokenizer.py

```
@@ -0,0 +1,103 @@
+        return tokens
+
+    def decodeBytes(self, tokens):
+        return b''.join(map(lambda i: self.idx2token[i], tokens))
+
+    def encode(self, src):
+        return self.encodeBytes(src.encode("utf-8"))
+
+    def decode(self, tokens):
+        try:
+            return self.decodeBytes(tokens).decode('utf-8')
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)